plugins/meta/aos-firewall/firewall/firewall.go

```diff
@@ -371,18 +440,45 @@ func (f *Firewall) formatIptablesRequest(chain *AccessChain) (chainFilters []ipt
 	adminParams := []iptablesRequest{
 		// Configure admin chains
 		{action: tableInsert, chain: "FORWARD", jump: forwardChainName},
+		{action: tableInsert, chain: "FORWARD", jump: forwardPortChainName},
 		{chain: "FORWARD", jump: outputChainName},
-		{chain: forwardChainName, dest: chain.Address.IP.String(), state: "NEW", jump: chain.Name},
+		{chain: forwardPortChainName, dest: chain.Address.IP.String(), state: "NEW", jump: chain.Name},
 		{chain: outputChainName, src: chain.Address.IP.String(), protocol: "icmp", jump: "DROP"},
 	}
 
 	chainFilters = append(chainFilters, adminParams...)
 
 	if !chain.HasInternetConnection {
 		adminParams = append(adminParams, iptablesRequest{
-			chain: outputChainName, src: chain.Address.IP.String(), state: "NEW", jump: "DROP"})
+			chain: outputChainName, src: chain.Address.IP.String(), state: "NEW", jump: "DROP",
+		})
+	} else {
+		ip, err := gateway.DiscoverInterface()
+		if err == nil {
```

Comment:
If the error occurs, what would happen? Nothing will work silently? IMHO error should be propagated to the top level (network manager) and this container should not start.

Reply:
Done